code/modules/shuttle/shuttles/dropship.dm

```diff
@@ -118,13 +118,12 @@
 	if(mode == SHUTTLE_CRASHED)
 		return
 
-	var/obj/docking_port/stationary/marine_dropship/dropzone = destination
-	if(mode == SHUTTLE_PREARRIVAL && !dropzone.landing_lights_on)
-		if(istype(destination, /obj/docking_port/stationary/marine_dropship))
-			dropzone.turn_on_landing_lights()
-		playsound(dropzone.return_center_turf(), landing_sound, 60, 0)
+	/*if(mode == SHUTTLE_PREARRIVAL && !destination.landing_lights_on)
+		if(istype(destination, /obj/docking_port/stationary))
+			destination.turn_on_landing_lights()
+		playsound(destination.return_center_turf(), landing_sound, 60, 0)
 		playsound(return_center_turf(), landing_sound, 60, 0)
-
+	*/
```

Comment:
Pretty sure this commenting out is why it's not working for Alamo or Normandy anymore, but the problem is this is an overriding cheat to make it work, and I can't figure out why this manages to work when it doesn't the way I did it.

Reply:
So if we're going to comment out this code; and you want to retain the old behavior where lights turn on when the shuttle is in `SHUTTLE_PREARRIVAL` mode, you're going to then need to add a proc thats like OnStartPrearrival or something.

We also need to double check the delayed landing lights still operate fine with just calling turn_on_landing_lights or if they were in some way utuilizing the repeated calls.